src/pymediainfo/__init__.py

```diff
@@ -298,13 +300,10 @@ def _get_library_paths(os_is_nt: bool) -> Tuple[str, ...]:
     @classmethod
     def _get_library(
         cls,
-        library_file: Optional[str] = None,
-    ) -> Tuple[Any, Any, str, Tuple[int, ...]]:
+        library_file: str | None = None,
```

Comment:
Apparently, this was added in Python 3.10:
> [PEP 604](https://peps.python.org/pep-0604/), Allow writing union types as X | Y
> https://docs.python.org/3/whatsnew/3.10.html

Will this cause any problems with Python 3.9? Should we drop Python 3.9 support? I'm hesitant as Debian 11 still has 3.9.

Reply:
It's not a problem because of `from __future__ import annotations`. It's not used at run time, so it works even with python 3.9 (or 3.8).